roles/ocp-route-redirect/tasks/redirect.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- k8s_facts:
+    api_version: route.openshift.io/v1
+    kind: Route
+    name: "{{ route_redirect_name }}"
+    namespace: "{{ route_redirect_namespace }}"
+  register: route
+
+- set_fact:
+    route: "http://{{ (route.resources | first).spec.host }}"
```

Comment:
Probably should the protocol configurable ... or maybe even require that the complete url is passed in and hence don't need this role to make a decision on protocol (i.e.: http v.s. https v.s. ftp, etc.)